app/models/user.rb

```diff
@@ -542,9 +543,15 @@ def add_renamed_at
     self.renamed_at = Time.current
   end
 
-   def log_change_if_login_was_edited
-     create_log_item(options = { action: ArchiveConfig.ACTION_RENAME, note: "Old Username: #{login_before_last_save}; New Username: #{login}" }) if saved_change_to_login?
-   end
+  def log_change_if_login_was_edited
+    create_log_item(action: ArchiveConfig.ACTION_RENAME, note: "Old Username: #{login_before_last_save}; New Username: #{login}") if saved_change_to_login?
+  end
+
+  def log_email_change
+    current_admin = User.current_user if User.current_user.is_a?(Admin)
+    note = current_admin ? "Change made by admin #{current_admin&.login}" : "Change made by user"
```

Comment:
I'm checking the notes for other actions/current notes for email changes and let's go with: 
- `"Change made by #{current_admin&.login}"` when it's an admin, for consistency with the role change note
- `System Generated` when it's the user, for consistency with the existing message (this is also consistent with user-made password changes)